src/m_obj.c

```diff
@@ -919,6 +920,27 @@ int obj_issignaloutlet(const t_object *x, int m)
     "MAINSIGNALIN" has been provided, in which case the object won't
     promote scalars correctly.  Nonetheless we provide it so that at least
     such a badly written object won't crash Pd. */
+
```

Comment:
This comment belongs to `obj_findsignalscalar` :) Just move the two functions above this comment.

Reply:
Or better: after `obj_findsignalscalar` so it matches your order in `s_imp.h`.